lib/hanami/cli/commands/app/db/migrate.rb

```diff
@@ -10,7 +10,7 @@ class Migrate < DB::Command
             desc "Migrates database"
 
             option :target, desc: "Target migration number", aliases: ["-t"]
-            option :dump, required: false, type: :boolean, default: true,
+            option :dump, required: false, type: :flag, default: true,
```

Comment:
I think we want this to be a `:boolean` still? Since this defaults to true, the thing that the user will be passing is actually `--no-dump`.

Can you double check we haven't mistakenly converted any other `:boolean` options with a default of `true`?

Reply:
Good call, fixed and confirmed there are no others.